packages/client/hmi-client/src/components/models/Model.vue

```diff
@@ -3,8 +3,14 @@
 		<header>
 			<div class="framework">{{ model?.framework }}</div>
 			<div class="header-and-buttons">
-				<h4 v-html="title" />
+				<h4>{{ title }}</h4>
 				<span v-if="isEditable">
+					<Button
+						v-if="isEditing"
+						@click="cancelEdit"
+						label="Cancel"
+						class="p-button-sm p-button-outlined"
+					/>
 					<Button
 						@click="toggleEditMode"
 						:label="isEditing ? 'Save model' : 'Edit model'"
```

Comment:
I wonder if we should reverse the buttons here, so that the toggle edit should be `Edit model` or `Cancel edit model` and the first conditional button is `Save edited model`.